### PR TITLE
Update: treepicker simple pure option to hide the svg icon

### PR DESCRIPTION
**Pull request by pphminions**
To have the option to remove the svg icon from the selected side of the tree picker

screen shot 2016-12-08 at 9 18 31 am

### CODE REVIEW

File: src/components/Main.jsx [hunk 1]
```diff
@@ -455,8 +455,10 @@ class AppComponent extends React.Component {
 
         <TreePickerSimplePure
           itemType={this.state.itemType}
+          hideIcon
           selectedNodes={[]}
           subtree={this.state.treePickerPureSubtree}
+          emptySelectedListText={<div><h><b>Choose items of interest</b></h></div>}
```
Comment (vinteo) [inline src/components/Main.jsx]: what is the `<h>` tag for?

Comment (pphminions) [inline src/components/Main.jsx]: oh a mistake @vinteo, will remove that, just wanted to show `emptySelectedListText` can contain anything :)

**Comment by pphminions** [comment]
@vinteo/ @omgaz - please merge

**Comment by omgaz** [comment]
Merging 🚀

**Pull request merged**